src/utils/issue.ts

```diff
@@ -183,13 +184,23 @@ export async function getAvailableOpenedPullRequests(context: Context, username:
       }
     }
 
-    if (reviews.length === 0 && (new Date().getTime() - new Date(openedPullRequest.created_at).getTime()) / (1000 * 60 * 60) >= reviewDelayTolerance) {
+    if (reviews.length === 0 && (new Date().getTime() - new Date(openedPullRequest.created_at).getTime()) / ms("1h") >= getTimeValue(reviewDelayTolerance)) {
```

Comment:
why is it converted to hours if `getTimeValue(reviewDelayTolerance)` returns milliseconds or am I missing something